From 4ee5ec8d924fc9812f768302ad755a76b433cad2 Mon Sep 17 00:00:00 2001
From: Tmplt <tmplt@dragons.rocks>
Date: Wed, 27 Jun 2018 21:13:22 +0200
Subject: [PATCH] cpuset fix shield

---
 cpuset/commands/set.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cpuset/commands/set.py b/cpuset/commands/set.py
index 9205a07..e299d70 100644
--- a/cpuset/commands/set.py
+++ b/cpuset/commands/set.py
@@ -410,8 +410,8 @@ def modify(name, cpuspec=None, memspec=None, cx=None, mx=None):
     log.debug('modifying cpuset "%s"', nset.name)
     if cpuspec: nset.cpus = cpuspec
     if memspec: nset.mems = memspec
-    if cx: nset.cpu_exclusive = cx
-    if mx: nset.mem_exclusive = mx
+    # if cx: nset.cpu_exclusive = cx
+    # if mx: nset.mem_exclusive = mx
 
 def active(name):
     """check that cpuset by name or cset is ready to be used"""
-- 
2.16.2

